prov/verbs/src/verbs_msg_ep.c

```diff
@@ -561,30 +560,36 @@ static inline int fi_ibv_poll_reap_unsig_cq(struct fi_ibv_msg_ep *ep)
 	return FI_SUCCESS;
 }
 
-/* WR must be filled out by now except for context */
 static inline ssize_t
-fi_ibv_send(struct fi_ibv_msg_ep *ep, struct ibv_send_wr *wr, void *context)
+fi_ibv_send_poll_cq_if_needed(struct fi_ibv_msg_ep *ep, struct ibv_send_wr *wr)
 {
-	struct fi_ibv_wre *wre = NULL;
 	int ret;
 
+try_again:
+	ret = FI_IBV_INVOKE_POST(send, send, ep->id->qp, wr, NULL);
```

Comment:
Can we kill this macro and just call separate functions for send and receive?  The only macro we need is one to convert the post_send/recv error number.

Reply:
Ok. I'm going to do that in the next PR